src/pages/AdminSettings/AdminMembers/MemberList/MemberTable.tsx

```diff
@@ -0,0 +1,237 @@
+import {
+  UseMutateFunction,
+  useMutation,
+  useQueryClient,
+} from '@tanstack/react-query'
+import {
+  createColumnHelper,
+  flexRender,
+  getCoreRowModel,
+  useReactTable,
+} from '@tanstack/react-table'
+import cs from 'classnames'
+import { useEffect, useMemo } from 'react'
+import { useInView } from 'react-intersection-observer'
+
+import { useLocationParams } from 'services/navigation'
+import {
+  SelfHostedSettings,
+  UserListOwner,
+  useSelfHostedSettings,
+  useSelfHostedUserList,
+} from 'services/selfHosted'
+import Api from 'shared/api'
+import Spinner from 'ui/Spinner'
+import Toggle from 'ui/Toggle'
+
+import 'ui/Table/Table.css'
+
+type MembersColumns = {
+  username: React.ReactNode
+  type: string
+  email: React.ReactNode
+  activationStatus: React.ReactNode
+}
+
+const columnHelper = createColumnHelper<MembersColumns>()
+
+const columns = [
+  columnHelper.accessor('username', {
+    header: 'User Name',
+    cell: ({ renderValue }) => renderValue(),
+  }),
+  columnHelper.accessor('type', {
+    header: () => <span className="flex grow flex-row text-center">Type</span>,
+    cell: ({ renderValue }) => renderValue(),
+  }),
+  columnHelper.accessor('email', {
+    header: 'Email',
+    cell: ({ renderValue }) => renderValue(),
+  }),
+  columnHelper.accessor('activationStatus', {
+    header: 'Activation Status',
+    cell: ({ renderValue }) => renderValue(),
+  }),
+]
+
+interface MutationArgs {
+  activated: boolean
+  ownerid: number
+}
+
+interface CreateTableArgs {
+  tableData: UserListOwner[]
+  seatData: SelfHostedSettings
+  mutate: UseMutateFunction<any, unknown, MutationArgs, unknown>
+}
+
+const createTable = ({ tableData, seatData, mutate }: CreateTableArgs) => {
+  return tableData?.map(
+    ({ ownerid, activated, email, isAdmin, name, username }) => {
+      let maxSeats = seatData?.seatsUsed === seatData?.seatsLimit
+      const disableToggle = maxSeats && !activated
+
+      return {
+        username: <p>{name || username}</p>,
+        type: isAdmin ? 'Admin' : 'Developer',
+        email: <p>{email}</p>,
+        activationStatus: (
+          <Toggle
+            dataMarketing="handle-members-activation"
+            label={activated ? 'Activated' : 'Non-Active'}
```

Comment:
Kinda similar thing here with Non-Active vs. Inactive vs. Not Activated. Maybe something to bring up to design / prod